src/ui/theme.c

```diff
@@ -6241,7 +6241,6 @@ meta_gtk_widget_get_font_desc (GtkWidget *widget,
   GtkStyleContext *style = gtk_widget_get_style_context (widget);
   GtkStateFlags state = gtk_widget_get_state_flags (widget);
   gtk_style_context_get(style, state, GTK_STYLE_PROPERTY_FONT, &font_desc, NULL);
-  font_desc = pango_font_description_copy (font_desc);
```

Comment:
Can you explain why removing this line?

Reply:
`gtk_style_context_get` creates a new object so there is no need to make a copy.

Reply:
In the example below used to test the `gtk_style_context_get` function, after getting the font description using `gtk_style_context_get`, it's freed using `pango_font_description_free` in the same function body.
```C
static void
test_basic_properties (void)
{
  GtkStyleContext *context;
  GtkWidgetPath *path;
  GdkRGBA *color;
  GdkRGBA *bg_color;
  PangoFontDescription *font;

  context = gtk_style_context_new ();
  path = gtk_widget_path_new ();
  gtk_style_context_set_path (context, path);
  gtk_widget_path_free (path);

  gtk_style_context_get (context, gtk_style_context_get_state (context),
                         "color", &color,
                         "background-color", &bg_color,
                         "font", &font,
                         NULL);
  g_assert (color != NULL);
  g_assert (bg_color != NULL);
  g_assert (font != NULL);

  gdk_rgba_free (color);
  gdk_rgba_free (bg_color);
  pango_font_description_free (font);

  g_object_unref (context);
}
```
https://github.com/GNOME/gtk/blob/1b77a4dba9b326f4d34f5c1e93b16db55b02ce27/testsuite/gtk/stylecontext.c#L270-L298